WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -0,0 +1,104 @@
+import UIKit
+
+final class CompliancePopoverCoordinator {
+    fileprivate enum Constants {
+        static let shouldShowCompliancePopup = "shouldShowCompliancePopup"
+    }
+
+    private let viewController: UIViewController
+    private let complianceService = ComplianceLocationService()
+    private let defaults: UserDefaults
+
+    init(viewController: UIViewController, defaults: UserDefaults = UserDefaults.standard) {
+        self.viewController = viewController
+        self.defaults = defaults
+    }
+
+    func presentIfNeeded() {
+        guard FeatureFlag.compliancePopover.enabled else {
+            return
+        }
+        complianceService.getIPCountryCode { [weak self] result in
+            switch result {
+            case .success(let countryCode):
+                guard let self, self.shouldShowPrivacyBanner(countryCode: countryCode) else {
+                    return
+                }
+                DispatchQueue.main.async {
+                    let complianceViewModel = CompliancePopoverViewModel(defaults: self.defaults)
+                    complianceViewModel.coordinator = self
+                    let complianceViewController = CompliancePopoverViewController(viewModel: complianceViewModel)
+                    let bottomSheetViewController = BottomSheetViewController(childViewController: complianceViewController, customHeaderSpacing: 0)
+
+                    bottomSheetViewController.show(from: self.viewController)
+                }
+            case .failure(let error):
+                ()
```

Comment:
This fixes an Xcode warning
```suggestion
            case .failure:
                break
```